.github/workflows/update-terms.yml

```diff
@@ -0,0 +1,81 @@
+
+name: Check for dwc terms
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - 'feature/events'
+#on:
+#  schedule:
+#    - cron: '0 0 1 * *'
```

Comment:
SHouldn't this be on `develop` ?

Reply:
added the feature/event to test this manually rather using cron event. will be replaced with cron schedule event.